src/Webapi/Webapi__Dom/Webapi__Dom__BeforeUnloadEvent.re

```diff
@@ -6,3 +6,8 @@ include Webapi__Dom__Event.Impl({ type nonrec t = t; });
 [@bs.new] external makeWithOptions : (string, Js.t({..})) => t = "BeforeUnloadEvent";
 
 [@bs.get] external returnValue : t => string = "";
+/**
+ * returnValue cannot be unset because BS cannot (afaik) bind `delete e['returnValue'];`
+ * Setting the value to undefined does not cause expected behavior.
+ */
+[@bs.set] external setReturnValue: (t, string) => unit = "returnValue";
```

Comment:
`return`Value` si actually defined on `Event`, which this inherits from. Could you move it there instead?

It;s also possible to write a function that does a `delete`, but you'll have to use `%raw`. Something like this should work I think:

```reason
let deleteReturnValue: t => unit = [%raw event => {|
  delete event['returnValue']
|}];
```

Reply:
Ah, indeed it is defined in [Event](https://developer.mozilla.org/en-US/docs/Web/API/Event/returnValue). I'll move it there and try adding deleting.